### PR TITLE
FT-02e: Add user-avatar and Random BG 

### PR DESCRIPTION
**Pull request by HasanMusaraf**
e. Add `user-avatar` component that takes user object and show the avatar in a circle. If avatar is missing, it shows first character of the first name with a random background color.

For random Background, have 26 colors and pick one color randomly.

### CODE REVIEW

File: app/components/user-avatar.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      '#C7D9A8', '#95D1F3', '#BFC5EE', '#E8BBC1'];
+  },
+
+  isWork: function () {
+    const charCodes = this.profile_char.charCodeAt();
+    return parseInt(charCodes, 10)
+
+  },
+
+  isClosing: computed('background', function () {
```
Comment (Shalini-Sivakumar) [inline app/components/user-avatar.js:23]: Add `avatar_colors` in the computed property triggers

File: app/styles/sub-styles/user-card.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
+.profile-icon {
+  width: 140px;
+  height: 140px;
+  border-radius: 50%;
+  display: flex;
```
Comment (Shalini-Sivakumar) [inline app/styles/sub-styles/user-card.scss:5]: Why you used flex property here? You have only one item inside this flex container.

File: app/components/user-avatar.js (hunk 1 above)
Comment (Shalini-Sivakumar) [inline app/components/user-avatar.js]: Add `avatar_colors` in the computed property triggers

File: app/templates/components/user-count.hbs [hunk 3]
```diff
@@ -1,2 +1,11 @@
 <h1>user count: {{ getUser.length}}</h1>
 
+<div class="avatar-wrapper">
+{{#each getUser as |user|}}
+
+{{#user-avatar user=user}}
+{{/user-avatar}}
+
+{{/each}}
```
Comment (Shalini-Sivakumar) [inline app/templates/components/user-count.hbs]: Add a proper indentation

**Review by Shalini-Sivakumar — CHANGES_REQUESTED** [review]

File: app/components/user-avatar.js (hunk 1 above)
Comment (HasanMusaraf) [reply app/components/user-avatar.js]: I have fixed this change.

File: app/templates/components/user-count.hbs (hunk 3 above)
Comment (HasanMusaraf) [reply app/templates/components/user-count.hbs]: I have fixed this change.

File: app/styles/sub-styles/user-card.scss (hunk 2 above)
Comment (HasanMusaraf) [reply app/styles/sub-styles/user-card.scss:5]: I have used flex property for positioning the content inside the avatar circle.

**Review by Shalini-Sivakumar — APPROVED** [review]

**Pull request merged**